modules/storages/spec/models/storage_spec.rb

```diff
@@ -77,13 +77,21 @@
   end
 
   describe '#destroy' do
-    let(:project) { create(project) }
+    let(:project) { create(:project) }
     let(:storage) { described_class.create(test_default_attributes) }
-    let(:project_storage) { Storages::ProjectStorage.create(project: project, storage: storage) }
+    let(:project_storage) { Storages::ProjectStorage.create(project: project, storage: storage, creator: test_default_creator) }
```

Comment:
Here you could use the factory that you have created in this PR.
```suggestion
    let(:project_storage) { create(:project_storage, project: project, storage: storage, creator: test_default_creator) }
```